src/components/Item/Item.js

```diff
@@ -9,20 +12,87 @@ import TotalPrice from './TotalPrice';
 import DiscountPrice from './DiscountPrice';
 import PriceBox from './PriceBox';
 import Wrapper from './Wrapper';
+import Details from './Details';
 
-function Item() {
+const getTotalPrice = ({ quantity, price }) => quantity * (price.regular - price.discount);
+
+const renderAttributes = function({ variants }) {
+  if (!variants) {
+    return null;
+  }
+
+  const size = variants.size || NO_SIZE;
+  const color = variants.color || NO_COLOR;
+
+  return (
+    <>
+      <Attribute>Size: {size}</Attribute>
+      <Attribute>Color: {color}</Attribute>
+    </>
+  );
+};
+
+const renderPrices = function({ price, quantity }) {
+  if (!price) {
+    return null;
+  }
+
+  return (
+    <PriceBox>
+      <DiscountPrice>Sale ${price.discount}</DiscountPrice>
+      <Price>Regular ${price.regular}</Price>
+      <TotalPrice>Total ${getTotalPrice({ quantity, price })}</TotalPrice>
```

Comment:
If you go this way, just pass two single arguments, instead of object. No point to wrap that with complex type.